services/common/rs_server_common/authentication.py

```diff
@@ -27,6 +30,17 @@
 # API key authentication using a header.
 APIKEY_SECURITY = APIKeyHeader(name=APIKEY_HEADER, scheme_name="API key passed in HTTP header", auto_error=True)
 
+# Look up table for stations
+STATIONS_AUTH_lut = {
```

Comment:
Could you use an all-uppercase name ? STATIONS_AUTH_LUT

Reply:
changed